app/components/media_component.html.erb

```diff
@@ -44,7 +44,7 @@
   <% end %>
 
   <% component.with_authorization_messages do %>
-    <div data-controller="auth-restriction" data-action="auth-denied@window->auth-restriction#displayMessage auth-stanford-restricted@window->auth-restriction#displayStanfordRestriction auth-success@window->auth-restriction#hideLoginPrompt" hidden="true" role="banner" aria-label="Access message">
+    <div data-controller="auth-restriction" data-action="auth-denied@window->auth-restriction#displayMessage auth-stanford-restricted@window->auth-restriction#displayStanfordRestriction auth-success@window->auth-restriction#hideLoginPrompt" hidden="true">
```

Comment:
Does this make an accessability violation where it now has a banner role with no visible content?

Reply:
It wasn't visible before anyway but no I double checked. Doesn't cause an accessibility error.